### ISSUE TITLE
Directory manager fail as dependency aren't installed

### ISSUE THREAD

**Issue by Benvii** [OpenPathView/OPV_Ansible#34]
Directory manager fails as dependency aren't installed.

```
● opv-dm.service - OPV Directory Manager
   Loaded: loaded (/etc/systemd/system/opv-dm.service; enabled; vendor preset: enabled)
   Active: inactive (dead) (Result: exit-code) since Mon 2018-05-14 18:50:00 UTC; 6min ago
  Process: 6456 ExecStart=/home/opv/bin/launch_dm.sh (code=exited, status=1/FAILURE)
 Main PID: 6456 (code=exited, status=1/FAILURE)

May 14 18:49:59 osfmmaster systemd[1]: opv-dm.service: Unit entered failed state.
May 14 18:49:59 osfmmaster systemd[1]: opv-dm.service: Failed with result 'exit-code'.
May 14 18:50:00 osfmmaster systemd[1]: opv-dm.service: Service hold-off time over, scheduling restart.
May 14 18:50:00 osfmmaster systemd[1]: Stopped OPV Directory Manager.
May 14 18:50:00 osfmmaster systemd[1]: opv-dm.service: Start request repeated too quickly.
May 14 18:50:00 osfmmaster systemd[1]: Failed to start OPV Directory Manager.

```

```
tail /home/opv/logs/directory_manager.log
Traceback (most recent call last):
  File "/home/opv/venvs/opv/bin/opv_dm_web.py", line 22, in <module>
    from opv_directorymanager import Webservice, default_config
  File "/home/opv/venvs/opv/lib/python3.5/site-packages/opv_directorymanager/__init__.py", line 21, in <module>
    from opv_directorymanager.storage_service import *
  File "/home/opv/venvs/opv/lib/python3.5/site-packages/opv_directorymanager/storage_service/__init__.py", line 20, in <module>
    from opv_directorymanager.storage_service.http import HTTP
  File "/home/opv/venvs/opv/lib/python3.5/site-packages/opv_directorymanager/storage_service/http.py", line 27, in <module>
    from gevent.wsgi import WSGIServer
ImportError: No module named 'gevent.wsgi'
```

I will try to investigate it tonight because we really need our Ansible script to be working again.

**Comment by Benvii** [comment]
Stop working on it as @Valdimus is also working on it.

**Comment by Valdimus** [comment]
Error was on the DirectoryManager, I tried to fix it with https://github.com/OpenPathView/DirectoryManager/pull/10

**Comment by Valdimus** [comment]
Fix with OpenPathView/DirectoryManager#10
